src/components/SubpageTabs.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", function() {
+  var menu, menuToggle, toggleButton;
+
+  menu = document.getElementById("tabs");
+  menuToggle = document.getElementById("dg_menu-toggle");
+  toggleButton = document.getElementById("dg_menu-trigger-btn");
+
+  toggleButton.addEventListener("click", function(c) {
```

Comment:
if you are going to use shorthand typeically `e` is used for an event. We have been using the convention `clickEvent` for this type of variable.

This is esepcially useful when you do something like this, which we do in react.

```
const handleClick = clickEvent => {
  // do cool stuff here
}

toggleButton.addEventListener("click", handleClick);
```

Reply:
Updated and committed. Went with `clickEvent` as the variable. Let me know if this looks ok. @martypowell